tests/NovelRT.Tests/Ecs/ConfiguratorTest.cpp

```diff
@@ -24,7 +24,7 @@ TEST(ConfiguratorTest, ConfiguratorCanProduceSystemSchedulerWithDefaultSystems)
 
 TEST(ConfiguratorTest, ConfiguratorCanProduceSystemSchedulerWithCustomComponentAndSystem)
 {
-    auto lambda = [](Timestamp delta, Catalogue catalogue) {
+    auto lambda = [](Timestamp, Catalogue catalogue) {
```

Comment:
This looks like a needless formatting change - this is just an example of what I was talking about there's quite a few instances of stuff like this and just wanted clarification

Reply:
It looks needless but compiler complains about `delta not being used`. I was not sure if it was wise to remove the parameter altogether and it was done this way elsewhere, so I get along.

Reply:
OH it removes the parameter??? I am so stupid lmao mobile makes this hard Ty for clarifying

Can you check other changes that might just look like formatting just to make sure? My eyesight is not the best on mobile

Reply:
I reviewed them again, just to be sure. It's all good. :)